www/js/survey/enketo-survey.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
when you added this, did you use `rm` followed by `add` or `mv`? `mv` retains the git history. Again, this is not critical for the current file which was pretty simple to begin with, but is very critical if you begin refactoring other files that have accumulated history.

Reply:
Not sure, I use vscode.

Reply:
Looks like in vscode you need to commit as soon as you rename and modify the file in a subsequent commit. If you do that, git will automatically detect the mv; otherwise it won't.
https://github.com/Microsoft/vscode/issues/6127